File: Dev/C++/Mascaret/src/VEHA/Kernel/BehavioralFeature.cpp
#include "VEHA/Kernel/BehavioralFeature.h"

namespace VEHA
{

BehavioralFeature::BehavioralFeature(const string& name,shared_ptr<Behavior> behavior)
:Feature(name),_method(behavior)
{
}

BehavioralFeature::~BehavioralFeature()
{
}

}

File: Dev/CS/Mascaret/Mascaret/VEHA/Kernel/BehavioralFeature.cs
using System;
using System.Collections.Generic;

namespace Mascaret
{
    public class BehavioralFeature : Feature
    {

        private Behavior method;
        public Behavior Method
        {
            get { return method; }
            set { method = value; }
        }

        public BehavioralFeature(string name, Behavior behavior)
            : base(name)
        {
            method = behavior;
        }

    }
}
